pkgs/development/python-modules/git-sim/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  installShellFiles,
+  gitpython,
+  manim,
+  opencv4,
+  typer,
+  pydantic,
+  fonttools,
+  git-dummy,
+}:
+
+buildPythonPackage rec {
```

Comment:
I think this package is an application but not a library.
Could you move it to `by-name/gi/git-sim` from `python-modules/git-sim`?
Also, please use `buildPythonApplication` instead.